src/main/java/frc/robot/subsystems/SuperStructure.java

```diff
@@ -22,75 +22,127 @@ public SuperStructure(
-            armPivot.moveToPosition(ArmPivot.PRESET_UP),
+            elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_FOUR_PRE_POS),
+            armPivot.moveToPosition(ArmPivot.CORAL_PRESET_UP),
             spinnyClaw.stop()),
-        armPivot.moveToPosition(ArmPivot.PRESET_PRE_L4),
+        armPivot.moveToPosition(ArmPivot.CORAL_PRESET_PRE_L4),
         Commands.waitUntil(score),
         Commands.parallel(
-            elevator.setLevel(ElevatorSubsystem.LEVEL_FOUR_POS),
-            armPivot.moveToPosition(ArmPivot.PRESET_L4)),
+            elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_FOUR_POS),
+            armPivot.moveToPosition(ArmPivot.CORAL_PRESET_L4)),
         spinnyClaw.holdExtakePower().withTimeout(0.2),
         stow());
   }
 
   public Command levelThree(BooleanSupplier score) {
     return Commands.sequence(
         Commands.parallel(
-            elevator.setLevel(ElevatorSubsystem.LEVEL_THREE_PRE_POS),
-            armPivot.moveToPosition(ArmPivot.PRESET_UP),
+            elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_THREE_PRE_POS),
+            armPivot.moveToPosition(ArmPivot.CORAL_PRESET_UP),
             spinnyClaw.stop()),
-        armPivot.moveToPosition(ArmPivot.PRESET_L2_L3),
+        armPivot.moveToPosition(ArmPivot.CORAL_PRESET_L2_L3),
         Commands.waitUntil(score),
-        elevator.setLevel(ElevatorSubsystem.LEVEL_THREE_POS),
+        elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_THREE_POS),
         spinnyClaw.holdExtakePower().withTimeout(0.15),
         stow());
   }
 
   public Command levelTwo(BooleanSupplier score) {
     return Commands.sequence(
         Commands.parallel(
-            elevator.setLevel(ElevatorSubsystem.LEVEL_TWO_PRE_POS),
-            armPivot.moveToPosition(ArmPivot.PRESET_UP),
+            elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_TWO_PRE_POS),
+            armPivot.moveToPosition(ArmPivot.CORAL_PRESET_UP),
             spinnyClaw.stop()),
-        armPivot.moveToPosition(ArmPivot.PRESET_L2_L3),
+        armPivot.moveToPosition(ArmPivot.CORAL_PRESET_L2_L3),
         Commands.waitUntil(score),
-        elevator.setLevel(ElevatorSubsystem.LEVEL_TWO_POS),
+        elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_TWO_POS),
         spinnyClaw.holdExtakePower().withTimeout(0.15),
         stow());
   }
 
   public Command levelOne(BooleanSupplier score) {
     return Commands.sequence(
         Commands.parallel(
-            elevator.setLevel(ElevatorSubsystem.LEVEL_ONE_POS),
-            armPivot.moveToPosition(ArmPivot.PRESET_L1),
+            elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_ONE_POS),
+            armPivot.moveToPosition(ArmPivot.CORAL_PRESET_L1),
             spinnyClaw.stop()),
         Commands.waitUntil(score),
-        elevator.setLevel(ElevatorSubsystem.LEVEL_TWO_POS),
+        elevator.setLevel(ElevatorSubsystem.CORAL_LEVEL_TWO_POS),
         spinnyClaw.holdExtakePower().withTimeout(0.15),
         stow());
   }
 
   public Command stow() {
```

Comment:
We should rename superstructure stow and intake to coralStow and coralIntake to be clear about which one it does since we have both coral and algae versions. (Renaming the levelOne through levelFour presets to include coral would also be nice)